### DISCUSSION TITLE
Tech Related Emojis On Github

### DISCUSSION THREAD

**Discussion by ml-sherpa** [community/community#129549] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

It would be great to have tech related emojis for commenting on PRs, issues etc, for example using 💎 when commenting on ruby code is strange. I should be using [https://github.com/buildkite/emojis/blob/main/img-buildkite-64/ruby.png](ruby.png).

There is a great list available at [buildkite/emojis](https://github.com/buildkite/emojis)

**Comment by NocturnalLament** [comment]
I would love this!
